src/Confluent.Kafka/Producer.cs

```diff
@@ -39,20 +39,20 @@ internal class Config
             public Action<string> statisticsHandler;
         }
 
-        private ISerializer<TKey> keySerializer;
-        private ISerializer<TValue> valueSerializer;
+        private SimpleSerializer<TKey> keySerializer;
```

Comment:
I dont like the word Simple here, it doesn't imply what is simple. Is it the type, or the serialization that is simple?

How about Base, Scalar, Primitive, Builtin, .. Serializer?

Reply:
I can leave the name as `Serializer`, it's not a huge deal. In the future, we could have `NoAllocSerializer` or `ZeroAllocSerializer` or `PipelineSerializer` or `SpanSerializer` or however we decide to implement it. Also, it's likely we may never implement this (certainly not soon - there are bigger priorities).

The primary message i was trying to get across here is that the serializer returns `byte[]` - is not as optimal as some more complex API that would allow allocations to be avoided in the serializer. It's also simple in the sense that it doesn't take a `SerializationContext`, but the delegate type to replace it also won't take a `SerializationContext`.

perhaps the word `basic` is an alternative? your suggestions seem to be getting more at the lack of `SerializationContext`.

The word `Simple` worked for me as it's a bit apologetic - when people take issue with it, it suggests we've though of that and something better is coming.

Reply:
I'll revert to `Serializer`.